shelley/chain-and-ledger/executable-spec/test/Generator/Utxo/QuickCheck.hs

```diff
@@ -159,22 +184,21 @@ genTxBody inputs outputs certs wdrls update fee slotWithTTL = do
              update
              Nothing -- TODO generate metadata
 
--- | Calculate the fee and distribute the remainder of the balance
--- to the given addresses (as transaction outputs)
+-- | Distribute the sum of `balance_` and `fee` over the addresses, return the
+-- sum of `fee` and the remainder of the equal distribution and the list ouf
+-- transaction outputs that cover the balance and fees.
+--
+-- The idea is to have an specified spending balance and fees that must be paid
+-- by the selected addresses.
 calcFeeAndOutputs
   :: Coin
   -> [Addr]
+  -> Coin
   -> (Coin, Seq TxOut)
-calcFeeAndOutputs balance_ addrs =
+calcFeeAndOutputs balance_ addrs fee =
```

Comment:
let's find a new name for this (since it's not calculating fees)

Reply:
In named it `calcOutputsFromBalance`